stable/wordpress/templates/deployment.yaml

```diff
@@ -13,6 +13,7 @@ spec:
     metadata:
       labels:
         app: {{ template "fullname" . }}
+        {{ template "mariadb.fullname" . }}-client: "true"
```

Comment:
I see, we use it here: https://github.com/kubernetes/charts/pull/1228/files#diff-d3bd2b28caf3d0673774d4321a1e3a66